python/tox.ini

```diff
@@ -1,14 +1,32 @@
 [tox]
-# This source package is built via `build-sdist.sh` as a workaround for the fact, that `setuptools_rust` doesn't support
-# local dependencies (`jsonschema` is in the parent directory)
-sdistsrc=dist/*.tar.gz
-envlist =
-  py{35,36,37,38}
+# Skip Source distribution build to allow each task to install it via pip
+# (workaround the fact that setup.py does not honor pyproject.toml)
+skipsdist = True
+
+envlist = py{35,36,37,38}
 
 [testenv]
+passenv =
+  # Env variable that would allow to build the jsonschema-rs library pointing to
+  # the jsonschema rust-library in the project (unreleased). Check setup.py for more details.
+  UNRELEASED_JSONSCHEMA_PATH
 deps =
   pytest
   pytest-benchmark
   hypothesis
 commands =
-  python -m pytest tests-py {posargs:}
+  pip install -e .  # Installing it within commands allow faster evn build (NOTE: uses debug rust build)
+  pip freeze
+
+[testenv:build-sdist]
+deps =
+  setuptools-rust
+commands =
+  python setup.py sdist
+
+[testenv:build-wheel]
+deps =
+  setuptools-rust
+  wheel
+commands =
+  python setup.py bdist_wheel
```

Comment:
Hm, it seems like the Linux wheels will not be manylinux compatible. Is it intentional? Or am I missing something?

Reply:
As far as I can tell the generated wheels should be compatible with systems installing manylinux compatible.

I'll do some extra search and ensure that manylinux is supported.

Please note that I had to disable windows tests as they are failing somehow ;)

Reply:
Today has been a very interesting learning around manylinux ... I was under the understanding that manylinux wheels are less preferable respect to `linux_x86_64` wheels as built with more "legacy" compatibilities but preferable due to the easier sharing and building.
Checking documentation online and pip implementation seems that the preference order is `manylinux2010_x86_64`, `manylinux1_x86_64` and then `linux`

The list is obtainable from
```
>>> from  pip._internal.models.target_python import get_supported
>>> get_supported()
```

I'm going to update the PR to build "regular" wheels for macos and windows, while opting for manylinux2010 wheels on linux

Reply:
Using manylinux2014 wheels and re-enabled windows tests :)